docs/intro-to-coalton.md

```diff
@@ -491,16 +491,14 @@ COALTON-USER> (type-of '/)
 ∀ :A :B. DIVIDABLE :A :B ⇒ (:A → :A → :B)
 ```
 
-Because of the generic nature of division, if you're computing some values at the REPL, "raw division" simply does not work.
+Division defaults to `Double-Float` operations when given `Integer` values. [see Instance Defaulting](#instance-defaulting).
 
 ```
 COALTON-USER> (coalton (/ 1 2))
-Failed to reduce context for DIVIDABLE INTEGER :A
-in COALTON
-   [Condition of type COALTON-IMPL/TYPECHECKER::COALTON-TYPE-ERROR-CONTEXT]
+0.5d0
 ```
 
-You have to constrain the result type of the `Dividable` instance. You can do this with the `the` operator. There are lots of `Dividable` instances made for you.
+In order to get a non-Double-Float result, you have to constrain the result type of the `Dividable` instance. You can do this with the `the` operator. There are lots of `Dividable` instances made for you.
```

Comment:
Show an example of this.

Re-word slightly, something like: "We can tell Coalton our constants are of another type by constraining them with `the`, or relying on type inference. The type must be an instance of `Dividable`" For example, ..."

Remember to read these things as if you're an absolute beginner trying to get work done.

Reply:
I believe there are two examples right below? One for `Single-Float` and another for `Fraction`

Reply:
Whoops, didn't see in the diff. Sorry!